JumpscaleLibs/clients/minio/MinioClient.py

```diff
@@ -0,0 +1,114 @@
+"""
+Minio Healing Client
+"""
+
+from Jumpscale import j
+import requests
+
+JSConfigClient = j.baseclasses.object_config
+
+
+class MinioClient(JSConfigClient):
```

Comment:
It's not minio client it's only for the `healing api`. So i suggest renaming to `MinioAdminClient` or `MinioHealingClient`